pandas/core/indexes/accessors.py

```diff
@@ -126,6 +126,26 @@ class DatetimeProperties(Properties):
     """
 
     def to_pydatetime(self):
+        """
+        Return DatetimeIndex as an object ndarray.
+
+        Returns
+        -------
+        ndarray of object dtype
+
+        See Also
+        --------
+        pandas.DatetimeIndex.to_pydatetime : Convert a Timestamp object to a native
```

Comment:
This is now referencing itself?

Reply:
check : https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DatetimeIndex.to_pydatetime.html
and 
https://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.dt.to_pydatetime.html

Reply:
Ah, sorry! Missed that this one was in `accessors.py`.